ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,35 @@
+# Week 7: Git Challenge
+
+Welcome to week 7 of software training! This week, you'll be completing a series of `git` and GitHub challenges. **To get started, click the "Submit new issue" at the bottom of this page!** From there, work through the rest of this checklist.
+
+If you need any help, we recommend these online resources:
+- https://help.github.com/articles/github-glossary/
+- https://try.github.io/levels/1/challenges/1
+- https://learngitbranching.js.org/
+- https://chris.beams.io/posts/git-commit/
+
+You can, of course, also flag down an instructor if you run into anything.
+
+## Your objectives
+- [ ] Star this repository
+- [ ] Make a personal copy of the repo that will live on your GitHub account. This allows you to freely make changes to the project without affecting the original. Your copy will remain associated with the original, allowing you to submit a pull request to the original author.
```

Comment:
Can we say fork here?

Reply:
I think I want to leave that up to them to figure out.....it will help them understand the difference between fork/clone/etc.

Reply:
While I see your point, I'm not sure this is phrased clearly enough for people to figure it out...

Reply:
I don't think that obsfucating the terminology helps them understand anything, fwiw.

Maybe you could just remove these steps completely, which would be a much more realistic scenario.

Reply:
Ideally we'd want them to look it up or rationalize it out themselves if they don't understand...we don't want them to just type words into the terminal from this list

Reply:
I think that there are multiple issues here:

1. The entire sentence is confusing. The actual intent of the sentence is "Fork the repo", but somehow it manages to be >200 chars.
2. The intent of the sentence is conflicting. The overall feel of the sentence feels like we're trying to hide information, but the latter half of the sentence feels like we're trying to help them out (by providing a very long description).

IMHO you should either:

Make it "easy": "Fork the repo on GitHub. [OPTIONAL explanation]"
Make it "hard": "Copy the repo on GitHub"
Make it realistic: omit this statement completely

Seeing as this step is required for literally everything else, I like "easy" the most, but I can see the merit of making it "realistic". I don't particularly (still) like "hard".